draft-ietf-acme-acme.md

```diff
@@ -1428,6 +1436,35 @@ over the same type, e.g., ["http-03", "http-05"].  Challenges within a type are
 testing the same capability of the domain owner, and it may not be possible to
 satisfy both at once. ]]
 
+## Validation of Wildcard Authorizations
```

Comment:
This should be a new challenge type or there should be some flag to indicate that this challenge method was used with an existing type.